src/app/CommandHandler.h

```diff
@@ -270,6 +300,28 @@ class CommandHandler
     CHIP_ERROR AddStatusInternal(const ConcreteCommandPath & aCommandPath, const Protocols::InteractionModel::Status aStatus,
                                  const Optional<ClusterStatus> & aClusterStatus);
 
+    /**
+     * Adds a data response.  The template parameter T is generally
+     * expected to be a ClusterName::Commands::CommandName::Type struct, but any
+     * object that can be encoded using the DataModel::Encode machinery and
+     * exposes the right command id will work.
+     *
+     * @param [in] aRequestCommandPath the concrete path of the command we are
```

Comment:
```suggestion
     *
     * If this function fails, it may leave our TLV buffer in an inconsistent
     * state.  Callers should snapshot as needed before calling this function,
     * and roll back as needed afterward.
     *
     * @param [in] aRequestCommandPath the concrete path of the command we are
```

Reply:
Resolved, but not detected by github.